JS/Lesson26/gulcin/app/script.js

```diff
@@ -3,12 +3,16 @@ const API_KEY = 'qCnxm3oi9NS9mzAkGmh7_4B6V3p6DJnyFs_wKfNQFF4';
 const button = document.querySelector('#fetch-button');
 
 button.addEventListener('click', function () {
-	fetch(`https://api.unsplash.com/photos/random?count=10&client_id=${API_KEY}`)
+
+	const imageDivs = document.querySelectorAll('.image');
+	imageDivs.forEach(div => div.remove());
```

Comment:
This works but there could be lots of images on the screen. Another approach would be set the content of `imageContainer` to an empty string, for example :)